src/SprykerEco/Zed/UnzerApi/Business/Api/Request/GetPaymentRequest.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEco\Zed\UnzerApi\Business\Api\Request;
+
+use Generated\Shared\Transfer\UnzerApiRequestTransfer;
+use Symfony\Component\HttpFoundation\Request;
+
+class GetPaymentRequest extends UnzerApiAbstractRequest implements UnzerApiRequestInterface
+{
+    /**
+     * @param \Generated\Shared\Transfer\UnzerApiRequestTransfer $unzerApiRequestTransfer
+     *
+     * @return string
+     */
+    public function getUrl(UnzerApiRequestTransfer $unzerApiRequestTransfer): string
+    {
+        return sprintf(
+            $this->unzerApiConfig->getUnzerApiGetPayment(),
+            $unzerApiRequestTransfer->getGetPaymentRequest()->getPaymentId()
```

Comment:
Please use `orFail` for methods inside cascade changing
e.g. `$unzerApiRequestTransfer-> getGetPaymentRequestOrFail()-> getPaymentId()`

Reply:
Fixed